### PR TITLE
Updating the wallet docs to use arweave.app as a starting point

### PR DESCRIPTION
**Pull request by DanMacDonald**
Updates to the main onboarding page, as well as FAQ and Cold Wallet instruction pages.
Also removes "Archive" instructions page.

### CODE REVIEW

**Review by angrymouse — APPROVED** [review]

File: SUMMARY.md [hunk 1]
```diff
@@ -11,21 +11,15 @@
 
 ## Wallets
 
-* [Arweave Web Extension Wallet](wallets/arweave-web-extension-wallet.md)
+* [Arweave.app Web Wallet](wallets/arweave-web-extension-wallet.md)
```
Comment (rosmcmahon) [inline SUMMARY.md:14]: suggestion: rename this file, otherwise the URL will be for the Arweave web extension

File: wallets/wallet-faq.md [hunk 2]
```diff
@@ -4,26 +4,26 @@ description: All questions surrounding wallets
...
+The following are wallets (developed by the Arweave community)  which are capable of storing AR tokens.
 
-## **How do I use the Arweave web extension wallet?**
+<table><thead><tr><th> </th><th data-type="checkbox">arweave.app</th><th data-type="checkbox">ArConnect</th><th data-type="checkbox">Finnie</th></tr></thead><tbody><tr><td>AR tokens</td><td>true</td><td>true</td><td>true</td></tr><tr><td>dApp permissions</td><td>true</td><td>true</td><td>true</td></tr><tr><td>Open Source</td><td>true</td><td>true</td><td>false</td></tr><tr><td>No Install</td><td>true</td><td>false</td><td>false</td></tr><tr><td>NFTs</td><td>false</td><td>true</td><td>true</td></tr><tr><td>PSTs</td><td>false</td><td>true</td><td>false</td></tr><tr><td>Page Archiving</td><td>false</td><td>true</td><td>true</td></tr></tbody></table>
+
+## **How do I use the arweave.app web wallet?**
 
 Lucky for you, we’ve created a step-by-step guide which can be [**found here**](https://docs.arweave.org/info/wallets/arweave-web-extension-wallet)**.**
 
 ## **Can I use the Arweave wallet as a cold wallet?**
```
Comment (rosmcmahon) [inline wallets/wallet-faq.md:17]: Arweave => arweave.app

File: wallets/wallet-faq.md [hunk 3]
```diff
@@ -4,26 +4,26 @@ description: All questions surrounding wallets
...
 
-Yes, you can find detailed instructions for how to securely generate an Arweave cold wallet with the official Arweave web extension [here](https://docs.arweave.org/info/wallets/generating-cold-wallet). 
+Yes, you can find detailed instructions for how to securely generate an Arweave cold wallet with the official Arweave web extension [here](https://docs.arweave.org/info/wallets/generating-cold-wallet).
```
Comment (rosmcmahon) [inline wallets/wallet-faq.md:19]: remove `official` and replace `Arweave web extension`

File: SUMMARY.md [hunk 4]
```diff
@@ -11,21 +11,15 @@
...
 * [Privacy Policy](wallets/privacy-policy.md)
 
-## Archiving
-
-* [Archiving Web Pages](archiving/archiving-web-pages.md)
-* [Step-by-Step Beginner's Archiving Guide](archiving/step-by-step-beginners-guide.md)
-
 ## Policies
 
 * [Privacy Policy](policies/privacy-policy.md)
```
Comment (rosmcmahon) [inline SUMMARY.md:21]: The privacy policy needs to be updated. It makes claims for the web extension.

File: SUMMARY.md (hunk 1 above)
Comment (rosmcmahon) [inline SUMMARY.md:14]: suggestion: rename this file, otherwise the URL will be for the Arweave web extension

**Review by rosmcmahon — CHANGES_REQUESTED** [review]
just a few typos and references that need changing.
in addition, if you have not already, you might want to make sure this is all displaying correctly in gitbook before merging.
e.g. use a page that is not connected to any menus to view it live

File: SUMMARY.md (hunk 1 above)
Comment (DanMacDonald) [reply SUMMARY.md:14]: I thought about it, I was worried about breaking existing bookmarks and references....

File: wallets/wallet-faq.md (hunk 2 above)
Comment (DanMacDonald) [reply wallets/wallet-faq.md]: image
How things look in gitbook ;)

File: SUMMARY.md (hunk 1 above)
Comment (rosmcmahon) [reply SUMMARY.md:14]: create a new page and a redirect from the old one? 
pretty standard to have an "update your bookmarks... ...you will be redirected in 5 seconds" page

Comment (rosmcmahon) [reply SUMMARY.md:14]: ....hold that thought

**Review by rosmcmahon — APPROVED** [review]
👍

**Pull request merged**